src/clojure_tictactoe/cli/input/input_getter.clj

```diff
@@ -0,0 +1,30 @@
+(ns clojure-tictactoe.cli.input.input-getter
+  (:require [clojure.string :as string]
+            [clojure-tictactoe.game.players :as players]))
+
+(defn get-user-input []
+  (let [user-input (string/trim (read-line))]
+    (if (= user-input "quit")
+      (. System exit 0)
+      user-input)))
+
+(defn continue-to-game []
+  (println "Press enter key to continue")
+  (get-user-input)
+  nil)
+
+(defn parse-move-input
+  ([input]
+   (let [formatted-input (read-string input)]
+     (when (and
```

Comment:
this `and` condition can be made more explicit through extracting it into a `let` binding that tells what it's doing. I think it's checking to see if the input is valid? If so, something along the lines of `let [move-is-valid (and (number? formatted-input) (< formatted-input 9))]`would be a good refactor.

Reply:
👍